deepchecks/base/display_suite.py

```diff
@@ -70,24 +75,41 @@ def display_suite_result(suite_name: str, results: List[Union[CheckResult, Check
             msg = result.exception.__class__.__name__ + ': ' + str(result.exception)
             name = result.check.name()
             others_table.append([name, msg, 1])
+        else:
```

Comment:
why fail all suite display if only single (or few) result(s) is problematic

Reply:
this is an internal function, if it receives something unexpected that means we have a bug in the code. WIthout `else` we just ignoring those bugs

Reply:
we have function `wrap_run` where we validate each return value of a check. I'll add over there assertion the return type is CheckResult, and than we won't have to worry about that

Reply:
ok, but it does not change anything/
writing `if-elif` statement without `else` is a bad practice
you never know in which part of the package your function will be used and what will be passed to it.

in the end, it is just one line of the code/